Source/DafnyRuntime/DafnyRuntime.py

```diff
@@ -39,10 +39,31 @@ def _tail_call():
     raise TailCall()
 
 class Seq(list):
+    def __init__(self, __iterable = None, isStr = False):
```

Comment:
The testing coverage for this is implicitly in `Hello.dfy`, I assume, which only worked before because we were just emitting a base Python string?

Reply:
Yes, we weren't doing anything with the strings other than passing them around, so it was fine.

Reply:
Good point, I'll try to add it in this one.